app/init/constants.php

```diff
@@ -49,6 +49,7 @@
 const APP_STORAGE_FUNCTIONS = '/storage/functions';
 const APP_STORAGE_BUILDS = '/storage/builds';
 const APP_STORAGE_CACHE = '/storage/cache';
+const APP_STORAGE_CSV_IMPORTS = '/storage/imports'; // Temporary storage for csv imports
```

Comment:
Can we simply call this imports? We don't really need to couple this with CSV, we can reuse it in the future for other formats.

Reply:
We can.
I was just worried that out of a random look, it shouldn't look unrelated to CSV. But I'll change and keep the comment anyways.